janusgraph-cql/.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this an eclipse specific file? should we be committing this to the repo, or should people be able to determine their own IDE configurations?